rules/windows/process_creation/proc_creation_win_screenconnect_remote_command_execution.yaml

```diff
@@ -0,0 +1,26 @@
+title: Remote Access Tool - ScreenConnect Remote Command Execution
+id: b1f73849-6329-4069-bc8f-78a604bb8b23
+status: experimental
+description: Detect the execution of system command via ScreenConnect RMM
+author: Ali Alwashali
+date: 2023/10/10
+tags:
+    - attack.execution
+    - attack.t1059.003
+logsource:
+    category: process_creation
+    product: windows
+detection:
+    selection_parent:
+        ParentImage|endswith: '\ScreenConnect.ClientService.exe'
+    selection_img:
+        - Image|endswith: '\cmd.exe'
+        - OriginalFileName: 'Cmd.Exe'
```

Comment:
Is CMD the only shell that can be used or is it a dynamic field that you can choose?

Reply:
No, it's selected by it's own, it's like a GUI field where you provide a command and it will take care of executing it for you